dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, NXP
+ * Copyright (c) 2020,2024 NXP
```

Comment:
space after comma and remove the (c)

Reply:
thanks for your reminder, I will update it.